frontend/kesaseteli/employer/src/components/application/steps/step2/accordions/EmploymentAccordion.tsx

```diff
@@ -0,0 +1,171 @@
+import DateInput from 'kesaseteli/employer/components/application/form/DateInput';
```

Comment:
represents accordion form for single employment 
![Uploading Screenshot 2021-09-20 at 3.42.46 PM.png…]()